test/Utilities/sparse_matrix.jl

```diff
@@ -203,6 +218,11 @@ function test_ScalarAffine_OneBased()
     @test A.rowval == [1, 1, 2, 1, 2]
     @test A.nzval == [1.0, 1.0, 2.0, 1.0, 3.0]
     @test A.colptr == [1, 2, 4, 6]
+    MOI.empty!(A)
```

Comment:
Remove the `final_touch` in the test above as well

Reply:
It's still useful to leave it in case we add something back (and to check that it really does nothing).